website/templates/project/registration_editor_templates.mako

```diff
@@ -154,4 +158,28 @@
     </div>
 </script>
 
+<script>
+ function setAllBoxes(value) {
```

Comment:
I need to pass `RegistrationEditor` as the viewModel to access contributors. I could alternatively put setAllBoxes on the prototype if that's better.